src/core-services/account/schemas/group.graphql

```diff
@@ -185,6 +197,16 @@ type UpdateAccountGroupPayload {
   group: Group
 }
 
+"The response from the `removeAccountFromGroup` mutation"
+type RemoveAccountFromGroupPayload {
+  "The same string you sent with the mutation params, for matching mutation calls with their responses"
+  clientMutationId: String
+
+  "The removed group"
+  group: Group
```

Comment:
Can add `!` to `Group` because we'll always return it if an error wasn't thrown.